src/python/m5/params.py

```diff
@@ -882,6 +882,9 @@ def __str__(self):
     def __eq__(self, other):
         return self.pc == other.get_pc() and self.count == other.get_count()
 
+    def __hash__(self):
+        return hash((int(self.pc), int(self.count)))
+
```

Comment:
Can you confirm that this change is needed?

Reply:
Your commenting on a commit, not the PR. This isn't a change, it's reverting a change in an earlier commit I think.